examples/Custom Widget - Hello World.ipynb

```diff
@@ -237,7 +237,7 @@
    "outputs": [],
    "source": [
     "%%javascript\n",
-    "require([\"widgets/js/widget\", \"widgets/js/manager\"], function(widget, manager){\n",
+    "require([\"nbextensions/widgets/widgets/js/widget\", \"nbextensions/widgets/widgets/js/manager\"], function(widget, manager){\n",
```

Comment:
Hm, strange, did you have to do this?  I thought nbextensions was aliased in our require.js config.